src/run.js

```diff
@@ -242,6 +243,27 @@ const minimalcss = async options => {
         .map(cssUrl => {
           const obj = cleaned[cssUrl]
           const cleanedAst = csstree.fromPlainObject(obj)
+          csstree.walk(obj, function(node) {
+            if (node.type === 'Url') {
+              var value = node.value;
+              let path
+              if (value.type === 'Raw') {
+                path = value.value;
+              } else {
+                path = value.value.substr(1, value.value.length - 2);
+              }
+              const absolute = /^https?:\/\/|^\/\//i
+              const root = /^\//
+              if (absolute.test(path)) {
+                // do nothing
+              } else if (root.test(path) /* && css path is the same domain as html document */) {
```

Comment:
need to fix this

Reply:
Did you want to complete that before I do a full code review?

Reply:
not sure how to handle this. WDYT?